scala/hello-world-http/build.sbt

```diff
@@ -0,0 +1,22 @@
+import sbt.Keys._
+import sbt._
+
+name := "hello-world-http"
+version := "1.0"
+scalaVersion := "2.13.12"
```

Comment:
isn't scala 2.13 legacy? shouldn't we provide an example for scala 3?

Reply:
Isn't scala 2.13 still used more? I can switch though. Probably it matter little here, if all dependencies support Scala 3

Reply:
IDK the data about usage of scala 2.13 but I would rather prefer to show an example with the latest Scala version. If someone is stuck on 2.13, they'll probably find a way to make it working.

Reply:
I'll merge this one and try adding a Scala 3 example as well

Reply:
> I'll merge this one and try adding a Scala 3 example as well

+1 for this approach.